### ISSUE TITLE
TypeError with production build

### ISSUE THREAD

**Issue by cjh1** [PAIR-code/umap-js#35]
```
Uncaught (in promise) TypeError: i.set is not a function
    at Function.value [as identity] (abstractMatrix.js:164)
    at a (index.js:99)
    at Object.e.exports [as default] (index.js:193)
    at S (umap.js:708)
    at t.initializeOptimization (umap.js:488)
    at t.initializeFit (umap.js:170)
    at t.<anonymous> (umap.js:129)
    at umap.js:32
    at Object.next (umap.js:13)
    at umap.js:7
```

This seems to be resolved by bumping the version of ml-levenberg-marquardt, see #34

**Comment by cannoneyed** [comment]
Hey just curious, what's your setup / usage? I've never seen this error before, and I'm curious if there's something deeper going on here...

**Comment by cjh1** [comment]
I am using it in a TypeScript React App, created using create-react-app, so the build configuration is pretty standard.

**Comment by cannoneyed** [comment]
Hey sorry, been neglectful of this issue for a while. Are you still noticing this issue?

**Comment by cjh1** [comment]
This was resolved by #34
